src/app/GitUI/Editor/FileViewer.cs

```diff
@@ -425,34 +425,43 @@ public ArgumentString GetExtraDiffArguments(bool isRangeDiff = false, bool isCom
             };
         }
 
-        public ArgumentString GetDifftasticArguments(bool isRangeDiff = false)
+        public (ArgumentString args, string extraCacheKey) GetDifftasticArguments(bool isRangeDiff = false)
```

Comment:
Use DelphiCase for returned tupels?
```suggestion
        public (ArgumentString Args, string ExtraCacheKey) GetDifftasticArguments(bool isRangeDiff = false)
```

Reply:
We have use lowercase in other tuples (about 20 usages so not much usage).

Reply:
We already have used both variants.
I am talking about return types. Decomposition (to dedicated local variables) is different.
One argument is: "When accessing tuple members, they should behave like any other public field/property, which is PascalCase."
The .NET Runtime team is also discussing about adding this to Roslyn analyzers.
What do you think, @RussKie?

Reply:
I _think_ there's an analyser for that. We just need to enable and enforce the code style via the .editorconfig. I proposed that some while ago in https://github.com/gitextensions/gitextensions/pull/8843 but it bogged down.

Reply:
I believe the current .NET style guide is to use the uppercase, but unless we enable analysers it's hard to enforce.